### ISSUE TITLE
Scripts for visualization

### ISSUE THREAD

**Issue by xunmeibuyue** [yunhao-zou/RawHDR#3]
Hello, can you provide the scripts for visualization? 
I have tried a simple pipeline of visualization, including wb, demosaic, ccm and gamma. Here is my visualization result.

Input:
![input](https://github.com/jackzou233/RawHDR/assets/55770298/90af32ef-d84f-4289-8f1c-1671e97baab9)

GT:
![gt](https://github.com/jackzou233/RawHDR/assets/55770298/952b222c-4ca7-4d70-b860-c91d1349c842)

I can hardly see any difference. So can you provide your scripts for visualization?

**Comment by WangJY06** [comment]
Agree. I’m sure that the provided GTs are still 0EV images but not the HDR images.

Input after tone mapping:
![7U6A8062_raw](https://github.com/jackzou233/RawHDR/assets/48671042/1757d3e4-c62c-4c71-95a6-87f3ad771f9e)

GT after tone mapping:
![7U6A8062_gt](https://github.com/jackzou233/RawHDR/assets/48671042/051a096c-4e4e-4f3d-ad90-c816c0eb21eb)

**Comment by Qirui-Y** [comment]
@WangJY06 Hello, can you share the code for the visualization?
